service/role.go

```diff
@@ -284,7 +285,7 @@ func (r *roleService) updateRoleToken(ctx context.Context, domain, role, proxyFo
 			proxyForPrincipal: proxyForPrincipal,
 			minExpiry:         minExpiry,
 			maxExpiry:         maxExpiry,
-		}, time.Unix(rt.ExpiryTime, 0).Sub(fastime.Now()))
+		}, time.Unix(rt.ExpiryTime, 0).Sub(expTimeDelta))
```

Comment:
no test cases to check the expire time in time🙈🤫

Reply:
??

Reply:
https://github.com/yahoojapan/athenz-authorizer/blob/3718419a43d56e6cd73462a00fd8501c3436f221/policy/daemon_test.go#L1344-L1350

something like above, check if the expiry is set with correct value inside the cache

Reply:
updated.